webapp/src/js/services/db-sync.js

```diff
@@ -32,17 +25,19 @@ angular.module('inboxServices').factory('DBSync',
 
     var readOnlyFilter = function(doc) {
       // don't try to replicate read only docs back to the server
-      return READ_ONLY_TYPES.indexOf(doc.type) === -1 &&
-             READ_ONLY_IDS.indexOf(doc._id) === -1 &&
-             doc._id.indexOf(DDOC_PREFIX) !== 0;
+      return (
+        READ_ONLY_TYPES.indexOf(doc.type) === -1 &&
+        READ_ONLY_IDS.indexOf(doc._id) === -1 &&
+        doc._id.indexOf(DDOC_PREFIX) !== 0
+      );
     };
 
     var getOptions = function(direction) {
       var options = {
         live: true,
         retry: true,
         heartbeat: 10000,
-        back_off_function: backOffFunction
+        back_off_function: backOffFunction,
```

Comment:
I am very cool with this becoming our standard.

Reply:
Yeah, that one is configurable and I turned it on for everywhere that's valid under es5.